Taskfile.yml

```diff
@@ -35,17 +57,25 @@ tasks:
         mdbook-toc
         wasm-bindgen-cli
         wasm-pack
+
+  install-brew:
+    # brew might be installed, so check first
+    status:
+      - which brew
+    cmds:
+      # https://stackoverflow.com/questions/58292862/how-to-install-homebrew-on-ubuntu-inside-docker-container
+      - useradd -m -s /bin/zsh linuxbrew
```

Comment:
Adding users is quite invasive, so it's fine to do in a Docker container, but not on someone's system. So I would suggest we put these in the Dockerfile, and revert to the "Please install brew" here. (and it needs to be compatible with MacOS here as below)